doc/getting-started/index.md

```diff
@@ -4,3 +4,4 @@
 * [Executing Processes with `run()`](run.md)
 * [Task Arguments](arguments.md)
 * [Using the Context](context.md)
+* [Remote execution](remote.md)
```

Comment:
Is it something we want in the **getting started** guide?

Reply:
I don't think so, but that's were it was added :upside_down_face:

Reply:
let's keep it for now